strongbox-security/strongbox-authentication-providers/strongbox-ldap-authentication-provider/src/main/java/org/carlspring/strongbox/authentication/api/ldap/LdapAuthenticationConfigurationManager.java

```diff
@@ -40,6 +41,9 @@
     @Inject
     private AuthenticationItemConfigurationManager authenticationItemConfigurationManager;
 
+    @Inject
+    private PasswordEncoder passwordEncoder;
```

Comment:
as @steve-todorov mentioned we shouldn't use common `BCryptPasswordEncoder` here and seems it should be replaced with `PasswordEncoderFactories.createDelegatingPasswordEncoder()` within project in general. @anki2189 can you please try it?

Reply:
Changes done as suggested.